PLAYBOOK.md

```diff
@@ -0,0 +1,19 @@
+# Playbook
+Common tasks and fixes to keep the system running
+
+## Image Link Broken
+[Open Trello issue](https://trello.com/c/030MTiMv/62-images-links-become-broken-in-staging)
+
+In staging, images are disappearing from the S3 bucket we store them in.  Until the root cause is discovered and the issue fixed, we will fix it by uploading an image to the S3 bucket with an identical name.
+
+1. Hover over the broken image or use the browser's developer tools and search for an `<img>` element to get the file name and path. In the staging environment, it should be something like `https://kcen-seattle-decidim-staging.s3.us-east-1.amazonaws.com/path/within/s3bucket/yourfile.jpg`
```

Comment:
I would describe the problem and solution as: 

If an image used as one of the Decidim system images is deleted from the S3 bucket, the Decidim admin panel will not allow removing that image from the page it's used on or making any other edits to the page. The fix is to use the Decidim admin panel to upload a new image to that page with the same name.

Reply:
Done - that's way easier than I thought it was.  Nice.